### DISCUSSION TITLE
Help with the format of the request

### DISCUSSION THREAD

**Discussion by kwello-alexhicks** [acheong08/ChatGPT#203] [Q&A]
Hi everyone - after getting everything installed correctly, and running from VS - no matter what I try I cannot get past:

Response not in correct format!
list index out of range

What is the correct format of the request, for tes/stream response?

https://imgur.com/a/tb84qs5

**Comment by Harry-Jing** [comment]
That cause by an incorrect response from api

**Reply by acheong08** [reply]
(I can't see the image as imagur is blocking my IP. Do you mind sharing it via GitHub?)

**Reply by Harry-Jing** [reply]
> (I can't see the image as imagur is blocking my IP. Do you mind sharing it via GitHub?)

![image](https://user-images.githubusercontent.com/69247286/206837578-78f9779b-cf16-4814-96b1-01b981f9b709.png)

**Reply by acheong08** [reply]
The two main possibilities here are expired token or rate limit. @kwello-alexhicks Do you mind running with `--text --debug` so that the error is clearer? The stream format can't show the full error.
